modules/README.md

```diff
@@ -1,53 +1,55 @@
-An overview of the contrib modules and a small explanation
```

Comment:
can you made atomic modification of the file? now it looks like the whole file was modified. I think, this is because of changed line endings. good editors should preserve original line endings